programming_examples/basic/vector_scalar_mul/vector_scalar_mul_placed.py

```diff
@@ -81,15 +84,19 @@ def core_body():
         tiles_to_trace = [ComputeTile2, ComputeTile2]
         if trace_size > 0:
             trace_utils.configure_packet_tracing_flow(tiles_to_trace, ShimTile)
+            trace_utils.configure_packet_ctrl_flow([ComputeTile2], CtrlShimTile)
+
+        trace_size_int32 = trace_size // 4
```

Comment:
Can you do math on dtypes instead of magic number 4?

Reply:
Good suggestion.